src/main/webapp/WEB-INF/templates/system/common/ice.ftl

```diff
@@ -4,7 +4,7 @@
 <!-- >>>>>>>>>>>>>>>>>>>>>>>>>>>>
 CrafterCMS Authoring Scripts
 >>>>>>>>>>>>>>>>>>>>>>>>>>>> -->
-<script defer src="/studio/static-assets/modules/editors/tinymce/v5/tinymce/tinymce.min.js"></script>
+<script defer src="/studio/static-assets/modules/editors/tinymce/v5.10.7/tinymce/tinymce.min.js"></script>
```

Comment:
This doesn't look like the latest path from the deps update PR

Reply:
yeah sorry, didn't push the latest change, done!